crates/bevy_reflect/src/enums/mod.rs

```diff
@@ -316,7 +316,7 @@ mod tests {
     #[test]
     fn enum_should_allow_generics() {
         #[derive(Reflect, Debug, PartialEq)]
-        enum TestEnum<T: FromReflect> {
+        enum TestEnum<T: FromReflect + TypePath> {
```

Comment:
Nit: This isn't needed anymore since we add the bounds, right?
```suggestion
        enum TestEnum<T: FromReflect> {
```

Reply:
Resolved